providers/base/units/networking/test-plan.pxu

```diff
@@ -75,6 +75,9 @@ unit: test plan
 _name: Manual networking tests for devices (after suspend)
 _description: Manual networking tests for devices (after suspend)
 include:
+    after-suspend-networking/info_device.*                       certification-status=blocker
```

Comment:
This one does not appear in the [list-bootstraped output](https://pastebin.canonical.com/p/6qpsY7ZhF8/) you provided. Not sure why since `after-suspend-networking-manual` is present in `client-cert-desktop-22-04-manual`.

Reply:
Since I didn't add also-after-suspend for this job, now It will appear as I've add the flag